pydis_site/templates/events/pages/code-jams/10/frameworks.html

```diff
@@ -0,0 +1,195 @@
+{% extends "events/base_sidebar.html" %}
+
+{% load static %}
+
+{% block title %}Summer Code Jam 2023{% endblock %}
+
+{% block breadcrumb %}
+    <li><a href="{% url "events:index" %}">Events</a></li>
+    <li><a href="{% url "events:page" path="code-jams" %}">Code Jams</a></li>
+    <li><a href="{% url "events:page" path="code-jams/10" %}">Code Jam 2023</a></li>
+    <li class="is-active"><a href="#">Approved Frameworks</a></li>
+{% endblock %}
+
+{% block event_content %}
+    <p>
+        Below is the list of approved frameworks that you can use for the code jam.
+        Please work with your team to choose a library (or multiple) that everyone can and want to develop with.
+        If there is a library not listed below that you think should be here, you're welcome to discuss it with the Events Team over in the Discord server.
+
+    </p>
+    <p>
+        Other libraries can be used to help build your application, but if their primary purpose is image manipulation or processing then they must be approved or from this list.
+        In particular, if you'd like to use a graph/plot creating library similar to MatPlotLib or a machine learning framework/model, double check with us before deciding to use it.
+    </p>
+
+    <h3 id="approved-frameworks"><a href="#approved-frameworks">Approved Frameworks</a></h3>
+
+    <div class="card mb-4">
+        <div class="card-content">
+            <div class="content">
+                <p class="subtitle">Pillow</p>
+                <p>Pillow is a fork of the Python Imaging Library (PIL). The Python Imaging Library adds image processing capabilities to your Python interpreter.
+
+                    This library provides extensive file format support, an efficient internal representation, and fairly powerful image processing capabilities.
```

Comment:
```suggestion
                <p>
                    Pillow is a fork of the Python Imaging Library (PIL). The Python Imaging Library adds image processing capabilities to your Python interpreter.

                    This library provides extensive file format support, an efficient internal representation, and fairly powerful image processing capabilities.
```
---
also fwiw the empty lines don't cause line breaks in the resultant page. I assume a `<br>` would do that if wanted, though fine either way.

Reply:
Yeah, I was lazy and ripped it straight from Pillow's github, formatting and all. I wasn't intending there to be linebreaks